.gitignore

```diff
@@ -9,3 +9,4 @@ yq
 openshift-clients/
 podman-remote/
 .sw[a-p]
+tmp_pv.yaml
```

Comment:
I'd still prefer to avoid temporary files as much as possible :)

Reply:
@cfergeau yes but I don't see options here :(